src/main/java/seedu/address/model/person/Person.java

```diff
@@ -109,7 +127,11 @@ public int getWeeksPresent() {
      * @return The total number of weeks in the period, typically 12 weeks.
      */
     public int getTotalWeeks() {
-        return this.attendance.getTotalWeeks();
+        Attendance stdatd = this.attendance.orElse(null);
+        if (stdatd != null) {
+            return stdatd.getTotalWeeks();
+        }
+        return -1;
```

Comment:
What do you think about using 0 as the default value instead to bring it in line with `getWeeksPresent()`'s default value of 0?